### ISSUE TITLE
When will the code be opened?

### ISSUE THREAD

**Issue by fujiawei0724** [JeffWang987/DriveDreamer#2]
Thanks for the great work. When will the code be opened? Thank you very much.

**Comment by abhigoku10** [comment]
@JeffWang987 @DriveDreamer  when shall be the code and pre trained models released ???? 
Thanks in advance

**Comment by JWFanggit** [comment]
Hello, your work is very fascinating, can you upload the code please, I've borrowed some of your modules for bounding box inputs and would like to run a simple comparison test using your methodology

**Comment by ykw3000** [comment]
+1

**Comment by silvercherry** [comment]
+1

**Comment by Bowa529** [comment]
+2

**Comment by 24werewolf** [comment]
＋１

**Comment by AlexiaJM** [comment]
+1

**Comment by zhukai242** [comment]
+1

**Comment by huiyiygy** [comment]
+1

**Comment by liangzid** [comment]
+1

**Comment by ZJWang9928** [comment]
+1

**Comment by WuZhongQing** [comment]
+1

**Comment by sugersyy** [comment]
+1

**Comment by JeffWang987** [comment]
Sorry for the late reply. We have released the research code for DriveDreamer. Additionally, we have reframed our code to provide greater flexibility and customization. Now, users can define data configurations, distributed training settings, model parameters, visualizations, and more directly within the configuration files. We hope this will facilitate easier experimentation and adaptation of our framework.
